dashboard/index.js

```diff
@@ -25,83 +24,79 @@ function Dashboard(options) {
   this.layoutModules.call(this);
   this.layoutAssets.call(this);
 
-  var self = this;
-  Object.defineProperty(global, 'console', {
-    value: mockConsole(function(value) {
-      self.setData({
-        type: 'log',
-        value: value
-      });
-    })
-  });
-
   this.screen.key(['escape', 'q', 'C-c'], function() {
     process.exit(0);
```

Comment:
I don't know the inner workings of `process.exit(0)`, but is it necessary to do `return process.exit(0);` to ensure the JS stops executing once a user presses one of those keys?